docs/release_notes.md

```diff
@@ -35,6 +36,15 @@
 - Fixed import of the constraint weight arrays on sparse arrays. *AdonisFX-1155*
 - Fixed a bug that caused geometry sliding constraints to not find valid triangles to slide on initialisation. *AdonisFX-1165*
 - Fixed paint flood operation not normalising the weights properly. *AdonisFX-1146*
+- Fixed a bug that occurred when data from the target mesh was missing. *AdonisFX-1183*
```

Comment:
`` - Fixed a bug that happened when reference mesh data was missing at AdnSkin deformer. *AdonisFX-1183```

Reply:
Done.

Reply:
Please, remove this one. We don't have to list fixes for bugs that where never exposed to the user.